src/ExternalApi/Isite/Domain/BaseIsiteObject.php

```diff
@@ -72,7 +72,7 @@ public function getFileId(): string
         return $this->fileId;
     }
 
-    public function getImage(): string
+    public function getImage(): IsiteImage
```

Comment:
I miss this one, this will throw and error when the image is null
```suggestion
    public function getImage(): ?IsiteImage
```